bbrconn_linux.go

```diff
@@ -38,21 +38,42 @@ func Wrap(conn net.Conn, onClose InfoCallback) (Conn, error) {
 	return &bbrconn{Conn: conn, tconn: tconn, onClose: onClose}, nil
 }
 
-func (conn *bbrconn) Info() (int, *tcpinfo.BBRInfo, error) {
+func (conn *bbrconn) BytesWritten() int {
+	return int(atomic.LoadUint64(&conn.bytesWritten))
+}
+
+func (conn *bbrconn) TCPInfo() (*tcpinfo.Info, error) {
+	var o tcpinfo.Info
+	b := make([]byte, 10000)
```

Comment:
After reducing the size to 200, `BenchmarkTCPInfo` turns from 4300 ns/op down to 2100 ns/op.